Bio/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Tom Hiddleston</title>
+    <link rel="stylesheet" type="text/css" href="css/styles.css">
```

Comment:
@zdburnett your pages is not displaying your styles because your folder is called `CSS` and your are requesting `css/styles.css` try to name all of your files and folders in lowercase to avoid these issues. 🔨

Reply:
Changed the name of the folder to lower case css.